install-development-requirements.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash -e
 
 printf "=================== INSTALL CONCENT_API REQUIREMENTS ====================\n"
-pip install -r concent_api/requirements.lock
+cd concent_api
+pip install -r requirements.lock
+cd ..
```

Comment:
This commit description says:
> Change requirements installation path in script 

A good commit description should mention why you're doing this because it's not obvious at a glance. I.e.:

```
install-development-requirements.sh: Run pip in the directory containing requirements file because now it contains relative paths
```

Also, this definitely needs a comment in the code because these two ways of running pip are usually equivalent. If only one works, it should be explained why.

Reply:
there were also relative paths before.. but comment is added

Reply:
Really? What else are we installing from a directory using `-e` with a relative path?